assets/css/woocommerce/woocommerce.scss

```
@@ -345,10 +345,16 @@ ul.products,
 		.woocommerce-loop-product__title,
 		.wc-block-grid__product-title,
 		.wc-block-grid__product-title > a {
-			font-size: 1rem;
 			font-weight: 400;
 			margin-bottom: ms(-3);
 		}
+		h2, // @todo Remove when WooCommerce 2.8 is live
+		h3, // @todo Remove when WooCommerce 2.8 is live
```

Should this be the following?

```suggestion
		h2, // @todo Remove when WooCommerce Blocks 2.8 is live
		h3, // @todo Remove when WooCommerce Blocks 2.8 is live
```

If so, since WooCommerce Blocks 2.8 has been out for a while, is this pull even needed anymore? Or do we still need to explicitly remove the `font-size: 1rem` as done above for the title selectors?

---

I'm not fully sure, I copied that from https://github.com/woocommerce/storefront/blob/f9a328f5105ea0efbd3ce7e9dc67d471997d54a8/assets/css/woocommerce/woocommerce.scss#L343-L349
That code is 4 years old, so I will go wager it was targeting WooCommerce 2.8, will delete it.

---

I think that if we remove the `h2` and `h3` selectors, we will need to add `.woocommerce-loop-category__title`, otherwise Categories titles would have a different style from Products.

So I would keep this as-is in this PR and open an issue or a follow-up PR with the goal to replace `h2` and `h3` with  `.woocommerce-loop-category__title`.

---

For reference:

* `h2` and `h3` selectors were added in 19a4c5f46787856e767b632fafee035798107b51
* The `.woocommerce-loop-product__title` class name was introduced in https://github.com/woocommerce/woocommerce/pull/11583.